Makefile

```diff
@@ -29,8 +29,8 @@ STATIC := $(STATIC)
 # depending on the context.
 COCKROACH_IMAGE :=
 
-RUN  := run
-GOPATH  := $(CURDIR)/_vendor:$(CURDIR)/../../../..
+RUN := run
+export GOPATH := $(CURDIR)/_vendor:$(CURDIR)/../../../..
```

Comment:
Yes, if the variable exists in the environment and is changed in make, make will re-export it.